cloud/linode/firewall/firewalls.go

```diff
@@ -205,10 +206,16 @@ func processACL(fwcreateOpts *linodego.FirewallCreateOptions, aclType, label, sv
 		ipv4chunks := chunkIPs(ipv4s)
 		for i, chunk := range ipv4chunks {
 			v4chunk := chunk
+			desc := fmt.Sprintf("Rule %d, Created by linode-ccm: %s, for %s", i, label, svcName)
+			if len(desc) > maxFirewallRuleDescLen {
+				newDesc := desc[0:maxFirewallRuleDescLen-3] + "..."
+				klog.Infof("Firewall rule description '%s' is too long. Stripping it to '%s'", desc, newDesc)
+				desc = newDesc
+			}
```

Comment:
can we break this out into a helper function?

Reply:
updated.